.circleci/config.yml

```diff
@@ -8,10 +8,14 @@ jobs:
       - checkout
       - run:
           name: Install dependencies
+          environment:
+            PATH: $HOME/.yarn/bin:$PATH
```

Comment:
Nope, that's making the build break because circle won't let you update env vars like that—removed!